lib/usageTracking.ts

```diff
@@ -85,12 +87,14 @@ export function trackCommandUsage(command, meta = {}, accountId) {
       );
       logger.debug('Sent usage tracking command event: %o', usageTrackingEvent);
     } catch (e) {
-      logger.debug('Usage tracking failed: %s', e.message);
+      if (e instanceof Error) {
```

Comment:
Do we have a standardized way to do this via util? Or is this the best way?

Reply:
I was looking to add something like that and I realized we have `debugError` in errorHandlers/index.ts

I think I'll just use that instead here since the error returned by that util _should_ have all the right info in it